src/pages/office/office.html

```diff
@@ -1,52 +1,279 @@
 <ion-header>
-    <ion-navbar>
-      <ion-title>Office</ion-title>
-      <ion-buttons end>
-          <button ion-button (click)="logout()">Logout</button>
-        </ion-buttons>
-    </ion-navbar>
-  </ion-header>
-  
-  <ion-content padding>
-    <ion-card>
-      <ion-card-header>
-        <ion-card-title>
-          Office
-        </ion-card-title>
-      </ion-card-header>
-      <ion-card-content>
-        <button ion-button (click)="popToRoot()">Journal Page</button>
-        <button ion-button navPush="TerminateTestPage">Terminate Test</button>
-      </ion-card-content>
-    </ion-card>
+  <ion-navbar hideBackButton>
+    <ion-title>Office</ion-title>
+    <ion-buttons end>
+      <button ion-button (click)="logout()">Logout</button>
+    </ion-buttons>
+  </ion-navbar>
+</ion-header>
 
-    <ion-card id="ETA" *ngIf="(pageState.etaFaults$ | async)">
-        <ion-card-content>
-          <ion-grid>
-            <ion-row align-items-start class="mes-data">
-              <ion-col col-34>
-                <h4 class="fault-heading">ETA</h4>
-              </ion-col>
-              <ion-col align-self-start>
-                <div id="etaFaults">{{pageState.etaFaults$ | async}}</div>
+<ion-content>
+        <ion-row class="mes-full-width-card" id="candidate-section">
+            <ion-col col-64><h2 id="candidate-name">{{pageState.candidateName$ | async}}</h2><h3  id="candidate-driver-number">{{pageState.candidateDriverNumber$ | async}}</h3></ion-col>
+            <ion-col></ion-col>
+            <ion-col col-auto style="float:right; text-align:right"><h2>{{pageState.startTime$ | async}}</h2><h3>test outcome: 
+              <span [class.mes-green]="pageState.isPassed$ | async" [class.mes-red]="!(pageState.isPassed$ | async)">{{pageState.testOutcome$ | async}}</span></h3>
```

Comment:
`ngClass` with a ternary would be more concise here.

Reply:
done